README.md

```diff
@@ -27,7 +27,7 @@ git clone https://github.com/sqlancer/sqlancer
 cd sqlancer
 mvn package -DskipTests
 cd target
```

Comment:
Since we use `cd target` here, specifying `./target/` below would break the quick-start instructions. Thank you for your PR nevertheless!